test/env.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var S = require('..');
+var $ = require('sanctuary-def');
+var type = require('sanctuary-type-identifiers');
+
+var eq = require('./internal/eq');
+
+
+var $Type = $.NullaryType(
```

Comment:
It would be nice if `sanctuary-def` exports `Type`, so I don't have to create it myself.